extension/src/popup/components/manageAssets/TrustlineError/index.tsx

```diff
@@ -70,9 +82,20 @@ const renderError = (
     case TRUSTLINE_ERROR_STATES.UNKNOWN_ERROR:
     default:
       return (
-        <InfoBlock variant={InfoBlock.variant.error}>
-          <p className="TrustlineError__title">An unknown error occurred.</p>
-        </InfoBlock>
+        <>
+          <InfoBlock variant={InfoBlock.variant.error}>
+            <div>
+              <p className="TrustlineError__title">
+                This transaction could not be completed.
+              </p>
+              {resultCodes ? (
+                <p className="TrustlineError__subtitle">
+                  Error code: {resultCodes}
```

Comment:
we're having some issues identifying *what* went wrong when a `change trust` fails. So, for now, we'll just show the `result_codes` from Horizon. In the future, we'll have a custom design for every error